sdk/avalon_sdk/fabric/fabric_work_order.py

```diff
@@ -0,0 +1,200 @@
+            return -1
+
+    def work_order_complete(self, work_order_id, work_order_response):
+        """
+        This function is called by the Worker Service to
+        complete a Work Order successfully or in error.
+        This API is for proxy model.
+        params
+            work_order_id is unique id to identify the work order request
+            work_order_response is the Work Order response data in string
+        Returns
+            errorCode is a result of operation.
+        """
+        if (self.__fabric_wrapper is not None):
+            if work_order_response is None:
+                logging.info("Work order response is empty")
+                return -1
+            params = []
+            params.append(work_order_id)
+            params.append(work_order_response)
+            txn_status = self.__fabric_wrapper.invoke_chaincode(
+                self.CHAIN_CODE,
+                'workOrderComplete',
+                params)
+            if txn_status is True:
+                return 0
+            else:
+                return -1
+        else:
+            logging.error(
+                "Fabric wrapper instance is not initialized")
+            return -1
+
+    def encryption_key_start(self, tag):
+        """
+        Function to initiate to set the encryption key of
+        worker.
+        """
+        logging.error("This API is not supported")
+        return None
+
+    def encryption_key_get(self, worker_id, requester_id,
+                           last_used_key_nonce=None, tag=None,
+                           signature_nonce=None,
+                           signature=None):
+        """
+        Function to worker's key from fabric block chain.
+        """
+        logging.error("This API is not supported")
+        return None
+
+    def encryption_key_set(self, worker_id, encryption_key,
+                           encryption_nonce, tag, signature):
+        """
+        Function to set worker's encryption key.
+        """
+        logging.error("This API is not supported")
+        return None
+
+    def get_work_order_submitted_event_handler(self, handler_func):
+        """
+        Function to start event handler loop for
+        workOrderSubmitted event
+        params:
+            handler_func is call back function name as string
+        returns:
+            event handler object
+        """
+        if (self.__fabric_wrapper is not None):
+            event_handler = self.__fabric_wrapper.get_event_handler(
+                self.WORK_ORDER_SUBMITTED_EVENT_NAME,
+                self.CHAIN_CODE,
+                handler_func
+            )
+            return event_handler
+        else:
+            logging.error(
+                "Fabric wrapper instance is not initialized")
+            return None
+
+    def get_work_order_completed_event_handler(self, handler_func):
+        """
+        Function to start event handler loop for
+        workOrderSubmitted event
```

Comment:
You probably intended workOrderCompleted

Reply:
Changed.